distribution/tools/keystore-cli/src/test/java/org/elasticsearch/common/settings/CreateKeyStoreCommandTests.java

```diff
@@ -53,48 +53,67 @@ public void testNotMatchingPasswords() throws Exception {
     }
 
     public void testDefaultNotPromptForPassword() throws Exception {
+        assumeFalse("Cannot open unprotected keystore on FIPS JVM", inFipsJvm());
         execute();
         Path configDir = env.configFile();
         assertNotNull(KeyStoreWrapper.load(configDir));
     }
 
     public void testPosix() throws Exception {
-        String password = randomFrom("", "keystorepassword");
-        terminal.addSecretInput(password);
-        terminal.addSecretInput(password);
-        execute();
```

Comment:
This actually never tested with the password, because it didn't pass `--password` on the command line.